### PR TITLE
fix duplicate alerts

### PR DESCRIPTION
**Pull request by elainearbaugh**
Move updating service from running after every status check run to a beat task.

Add tests for update_service (w/ CELERY_ALWAYS_EAGER)

### CODE REVIEW

**Comment by elainearbaugh** [comment]
trigger build

**Comment by elainearbaugh** [comment]
tested locally and services are being updated on schedule

File: cabot/cabotapp/models.py [hunk 1]
```diff
@@ -473,12 +472,6 @@ def duplicate(self, inst_set=(), serv_set=()):
             linked.status_checks.add(new_check)
         return new_check.pk
 
-    def update_related_services(self):
```
Comment (elainearbaugh) [inline cabot/cabotapp/models.py]: no longer used

File: cabot/cabotapp/models.py [hunk 2]
```diff
@@ -459,7 +459,6 @@ def save(self, *args, **kwargs):
             self.cached_health = ''
             self.calculated_status = Service.CALCULATED_PASSING_STATUS
         ret = super(StatusCheck, self).save(*args, **kwargs)
-        self.update_related_services()
```
Comment (elainearbaugh) [inline cabot/cabotapp/models.py]: With many checks running at once, updating services after every check run (and therefore likely multiple at a time) is unnecessary and can overload celery

**Comment by mahendra** [comment]
Looks good to me. As part of the deploy, please take of the following
1. Set up `cloudwatch -> pagerduty` alerts for the new task (so that we are woken up if the job is not running)
2. Do a sanity check to ensure that the said alerts are going to infra oncall.

Also, good call to move this out ..

**Pull request merged**